client/src/components/UpLoadImages/style.css

```diff
@@ -0,0 +1,40 @@
+
+input[type="file"] {
+  display: none;
+}
+.custom-file-upload {
+  display: flex;
+  align-content: center;
+  align-items: center;
+  height: 60px;
+  width: 150px;
+  border: 1px solid #ccc;
+  padding: 6px 12px;
+  cursor: pointer;
+  margin: 1.1rem;
+}
+.contaner {
```

Comment:
Did you intend to miss spell `container`,

Reply:
yes, I did